mantidproject/mantid
# Crystal field IntensityScaling improvements and bug fixes

**Description of work.**

A new section describing IntensityScaling was added. It provides general information on intensity scaling for the CrystalField interface and also different examples and caveats with regard to the 4 general use-cases (single-site or multi-site with either single or multiple spectra).

There were a few minor bugs that prevented passing on of the IntensityScaling values to the excitation calculations. 

Although the syntax for the multi-spectra case was not changed as requested in #[30855](https://github.com/mantidproject/mantid/issues/30855) a few warnings and notices were added to make it easier for the user to be aware of the current use-case.

**Report to:** @mducle

**To test:**

Check the documentation and make sure that the provided examples make sense. 

Fixes #[30855](https://github.com/mantidproject/mantid/issues/30855)

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
